package.json

```diff
@@ -17,7 +17,7 @@
     "@ckeditor/ckeditor5-essentials": "^1.0.0-alpha.2",
     "@ckeditor/ckeditor5-heading": "vladikoff/ckeditor5-heading#short-heading",
     "@ckeditor/ckeditor5-list": "^1.0.0-alpha.2",
-    "@ckeditor/ckeditor5-markdown-gfm": "^1.0.0-alpha.2",
+    "@ckeditor/ckeditor5-markdown-gfm": "vladikoff/ckeditor5-markdown-gfm#data-as-html",
```

Comment:
Can we make this part of this repository?

Reply:
i will try!!

Reply:
Fixed by a function override instead of using the fork

Reply:
So shall we get back to `^1.0.0-alpha.2`?